device_info.go

```diff
@@ -81,9 +81,18 @@ func parseDeviceLong(line string) (*DeviceInfo, error) {
 			"malformed device line, expected at least 5 fields but found %d", len(fields))
 	}
 
+	if strings.Contains(line, "no permissions (user in plugdev group; are your udev rules wrong?)") {
+	} else {
+	}
+
 	state, err := parseDeviceState(fields[1])
 	if err != nil {
-		return nil, err
+		if strings.Contains(line, "no permissions (user in plugdev group; are your udev rules wrong?)") {
```

Comment:
why are we doing this inside the `err` check?  can we do this before `parseDeviceState()` call?

Reply:
moved to before `parseDeviceState`